test/CBF/CBF.jl

```diff
@@ -0,0 +1,322 @@
+
+import Base
+Base.isapprox(f::MOI.VectorOfVariables, g::MOI.VectorAffineFunction{Float64};
+    kwargs...) = isapprox(MOI.VectorAffineFunction{Float64}(f), g)
```

Comment:
What are other work-arounds for this? Why is it necessary?

Reply:
this isapprox is used only at https://github.com/JuliaOpt/MathOptInterface.jl/blob/374695cb096fccf3b17696855260abd7c525e873/src/Utilities/functions.jl#L420
because the CBF reader cannot detect VectorOfVariables constraints (all constraints are parsed as VectorAffineFunction).